addons/account/views/account_move_views.xml

```diff
@@ -1054,6 +1054,7 @@
                                 <group id="other_tab_group">
                                     <group string="Invoice"
                                            name="sale_info_group"
+<<<<<<< HEAD
                                            attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund'))]}">
                                         <label for="ref" string="Customer Reference" />
                                         <field name="ref" nolabel="1"/>
```

Comment:
Just discard everything in this file and add this patch:

```suggestion
                                        <field name="ref" nolabel="1"/>
                                        <field name="user_id" invisible="1" force_save="1"/>
```

It's the same as in https://github.com/odoo/odoo/pull/71328.

I don't have write permissions in the branch, so you'll have to do it.